deploy-bridge/README.md

```diff
@@ -0,0 +1,71 @@
+# E2E tests
```

Comment:
Is this readme related?

Reply:
I see there are a couple of files that are redundant and old. I will clean that.